File: server-scala-play-server/app/model/RoutingResponse.scala
package model

import play.api.libs.json._
import java.time.OffsetDateTime

/**
  * Represents the Swagger definition for RoutingResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class RoutingResponse(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  items: Option[List[LoadResponse]],
  vehicles: Option[List[VehicleResponse]],
  drivers: Option[List[DriverResponse]],
  config: Option[ConfigRoutingRequest],
  error: Option[String],
  debug: Option[List[String]],
  startTime: Option[OffsetDateTime],
  endTime: Option[OffsetDateTime],
  responses: Option[List[RoutingResponse]],
  orignalRequest: Option[String],
  generated: Option[Boolean],
  itineraries: Option[List[ItineraryResponse]],
  score: Option[Long],
  loadSize: Option[Int],
  runtime: Option[Long],
  earliestPickupWindow: Option[Long],
  latestPickupWindow: Option[Long],
  returning: Option[String]
)

object RoutingResponse {
  implicit lazy val routingResponseJsonFormat: Format[RoutingResponse] = Json.format[RoutingResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/RoutingResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class RoutingResponse 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public List<LoadResponse> Items { get; set; }
    public List<VehicleResponse> Vehicles { get; set; }
    public List<DriverResponse> Drivers { get; set; }
    public ConfigRoutingRequest Config { get; set; }
    public string Error { get; set; }
    public List<string> Debug { get; set; }
    public DateTime StartTime { get; set; }
    public DateTime EndTime { get; set; }
    public List<RoutingResponse> Responses { get; set; }
    public string OrignalRequest { get; set; }
    public bool Generated { get; set; }
    public List<ItineraryResponse> Itineraries { get; set; }
    public long Score { get; set; }
    public int LoadSize { get; set; }
    public long Runtime { get; set; }
    public long EarliestPickupWindow { get; set; }
    public long LatestPickupWindow { get; set; }
    public string Returning { get; set; }
}


